cron/releasetest/k8s/controller.yaml

```diff
@@ -40,4 +65,10 @@ spec:
                 memory: 1Gi
               requests:
                 memory: 1Gi
-          restartPolicy: OnFailure
+          - name: worker-update
+            image: bitnami/kubectl
```

Comment:
I would pin the SHA for this.

Reply:
Could you tell me how I would do that? Looks like this is their release version commit - https://github.com/bitnami/bitnami-docker-kubectl/commit/c1bf66c18a2dde1c3ad7f070f3fad15611862e6d.

So the commit SHA `c1bf66c18a2dde1c3ad7f070f3fad15611862e6d` would be what we pin it to?

Reply:
Nvm, used `docker inspect` to get it.